lib/deck.rb

```diff
@@ -0,0 +1,35 @@
+require 'pry'
+require './lib/card'
+class Deck
+  attr_accessor :cards, :cards_in_category
```

Comment:
since you already have a cards_in_category method, you do not need to have an attr_Reader. We use attr_readers when we want to change attributes (instance variables in our initialize method) into methods so that we can read those attributes on that object outside of the class itself.